django-stubs/views/generic/base.pyi

```diff
@@ -24,7 +25,7 @@ class TemplateResponseMixin:
     content_type: Optional[str] = ...
     request: http.HttpRequest = ...
     def render_to_response(self, context: Dict[str, Any], **response_kwargs: Any) -> http.HttpResponse: ...
-    def get_template_names(self) -> List[str]: ...
+    def get_template_names(self) -> Union[Sequence[str], Template, str]: ...
```

Comment:
This one looks strange to me on the first sight. What the docs say about the return type?

Reply:
I was to eager to not change my code when I wrote this. Turns out some child classes rely on the fact that it returns a list. I'll remove this change.